packages/auth/lib/modular/index.d.ts

```diff
@@ -659,6 +689,273 @@ export function getAdditionalUserInfo(
  * Returns the custom auth domain for the auth instance.
  *
  * @param auth - The Auth instance.
- * @returns A promise that resolves with the custom auth domain.
+ * @returns {string} A promise that resolves with the custom auth domain.
  */
 export function getCustomAuthDomain(auth: Auth): Promise<string>;
+
+/**
+ * Apple Authentication Provider.
+ * Not in JS Docs
+ *
+ */
+export class AppleAuthProvider {
```

Comment:
Are we sure the provider types that already exist on RNFB can't be imported out of `lib/index.d.ts` and reused?